<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTRecMeasLoopParamDlg</class>
 <widget class="QDialog" name="CTRecMeasLoopParamDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainButtonLayout">
    <item>
     <widget class="QPushButton" name="CancelButton">
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OKButton">
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="SchemeLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>20</y>
     <width>615</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="CoefGroupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>170</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Коэффициенты деления измерительной цепи</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="CoefLayout">
     <item>
      <widget class="QLabel" name="k1Label">
       <property name="text">
        <string>k₁ = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="k1SpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="k2Label">
       <property name="text">
        <string>k₂ = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="k2SpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="k3Label">
       <property name="text">
        <string>k₃ = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="k3SpinBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="EqCoefBanner">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>331</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Эквивалентный коэффициент деления:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EqCoefLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>k₁₂₃ = </string>
   </property>
  </widget>
  <widget class="QLabel" name="EqCoefValue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
